populate/src/main/resources/seed/portals/demo/studies/heartdemo/surveys/preEnroll.json

```diff
@@ -80,6 +88,27 @@
             ]
           }
         ]
+      },
+      {
+        "elements": [
+          {
+            "type": "checkbox",
```

Comment:
this should be a radio button for now--we're not yet supporting enrolling both as subjects, right?  That will also simplify the parsing -- the value will just come in as the `booleanValue` on the question

Reply:
Right now in Pepper both approaches exist. In CMI these are check boxes and if they choose both, they will first sign up as normal users and they can later add their governed user. But in AT it's as you described, radio buttons and only supporting one kind of enrollment. I will change it to radio buttons, it shouldn't be hard to change it later anyways